src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
+    while (computerNumber.length < 3){
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if(!computerNumber.includes(number))
+        computerNumber.push(number);
+    }
+    return computerNumber;
+  }
+
+  async getUserInput(){
+    const userInput = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요 : ");
+    const validatedUserInput = this.checkInputValidation(userInput);
+
+    return validatedUserInput;
+  }
+
+  checkInputValidation(userInputNumber){
+    const userInputNumberArr = userInputNumber.split('').map(Number);
+
+    if(userInputNumber.length !== 3) throw new Error("[ERROR] 3자리 숫자가 아닙니다.");
```

Comment:
3 과 같은 매직넘버는 상수로 관리하면 추후 유지보수에 이점이 있을 것 같아요!
예를 들어 현재는 게임 규칙을 4자리 수로 변경할 경우 모든 3을 변경해야하지만 
상수로 관리하면 상수 값 하나만 변경하면 됩니다!

```suggestion
    if(userInputNumber.length !== MAX_LENGTH) throw new Error("[ERROR] 3자리 숫자가 아닙니다.");
```